fobi_custom/plugins/pldp/form_elements/fields/age_multiple/forms.py

```diff
@@ -0,0 +1,61 @@
+from django import forms
+
+from fobi.base import BasePluginForm
+from pldp.forms import AGE_BASIC_CHOICES, AGE_DETAILED_CHOICES, \
+                       AGE_COMPLEX_CHOICES
+
+import uuid
+
+class PLDPAgeMultipleForm(forms.Form, BasePluginForm):
+    """PLDPAgeMultipleForm."""
+
+    plugin_data_fields = [
+        ("label", "How many people do you see between the ages of 0-14?"),
+        ("name", uuid.uuid4()),
+        ("age range", ""),
+        ("required", False),
+    ]
+
+    structured_choices = tuple((('basic', tuple(AGE_BASIC_CHOICES)),
+                   ('detailed', tuple(AGE_DETAILED_CHOICES)),
+                   ('complex', tuple(AGE_COMPLEX_CHOICES))))
+
+    label = forms.CharField(label="Label",
+                            required=True,
+                            help_text="Use this survey element to count the \
+                            number of people of a specific age range. Change \
+                            the question to match your selection from the \
+                            dropdown below.")
+    name = forms.CharField(required=True, widget=forms.widgets.HiddenInput())
+    age = forms.ChoiceField(choices=structured_choices,
```

Comment:
I think this provides a good initial solution! (And thank you for the explanatory help text.) We definitely have enough to consider in conversation with UCD, who will help us iron out the details (I'm sure).